src/equations/compressible_euler_1d.jl

```diff
@@ -343,25 +347,25 @@ Entropy conserving two-point flux by
     # Unpack left and right state
     rho_ll, v1_ll, p_ll = cons2prim(u_ll, equations)
     rho_rr, v1_rr, p_rr = cons2prim(u_rr, equations)
-    beta_ll = 0.5 * rho_ll / p_ll
-    beta_rr = 0.5 * rho_rr / p_rr
-    specific_kin_ll = 0.5 * (v1_ll^2)
-    specific_kin_rr = 0.5 * (v1_rr^2)
+    beta_ll = 0.5f0 * rho_ll / p_ll
+    beta_rr = 0.5f0 * rho_rr / p_rr
+    specific_kin_ll = 0.5f0 * (v1_ll^2)
+    specific_kin_rr = 0.5f0 * (v1_rr^2)
 
     # Compute the necessary mean values
-    rho_avg = 0.5 * (rho_ll + rho_rr)
-    rho_mean = ln_mean(rho_ll, rho_rr)
-    beta_mean = ln_mean(beta_ll, beta_rr)
-    beta_avg = 0.5 * (beta_ll + beta_rr)
-    v1_avg = 0.5 * (v1_ll + v1_rr)
-    p_mean = 0.5 * rho_avg / beta_avg
+    rho_avg = 0.5f0 * (rho_ll + rho_rr)
+    rho_mean = ln_mean(rho_ll, rho_rr) # TODO: Modify ln_mean() from math.jl
+    beta_mean = ln_mean(beta_ll, beta_rr) # TODO: Modify ln_mean() from math.jl
```

Comment:
I added this to the list of tasks in https://github.com/trixi-framework/Trixi.jl/issues/591 so that you can remove the TODO notes here

Reply:
What about keeping the list of tasks by myself on this PR page? That way, I can check and modify it myself if more functions come in.

Reply:
That would also be okay. But I prefer having multiple smaller PRs since that makes it much easier to review and we get the (partial) improvements earlier into `main`. Thus, you can also open a new issue or add a new post with your list of TODO notes to the other issue linked above. You should also keep a list of files that need to be edited there.
If you agree, I would like to get a review from @sloede and then merge this PR with changes to the files you have already edited so that we get a nice template for similar changes to the other files.

Reply:
I have already requested a review from Michael. 

Yes I would agree your ideas that it is easier to manage improvements by multiple PRs. But I was wondering whether merging these changes into main separately will cause any problems. For example, we merge the improvements for equations first and then for other auxiliary files, will it cause any potential problems if any other PRs or tests are going to be done between these merges? 

If you treat it as a safe way I will soon open other smaller PRs as you proposed.

Reply:
Let's do it like this: You can push more changes here until @sloede has some time to review the changes. Then, we iterate this PR until all comments are resolved and the approach looks good to us. Next, we merge this PR - new changes can be done in other PRs.

Reply:
Thanks for suggestions! I have added the first helper PR. See above